guide/message_transformation/message_transformation.bal

```diff
@@ -0,0 +1,370 @@
+//    name:"ballerina-guides-message-transformation-service",
+//    baseImage:"ballerina/ballerina-platform:0.980.0",
+//    copyFiles:[{target:"/ballerina/runtime/bre/lib",
+//        source:"/home/saneth/Documents/ballerina/mysql-connector-java-5.1.46.jar"}]
+//}
+
+//@docker:Config {
+//    registry:"ballerina.guides.io",
+//    name:"message_transformation",
+//    tag:"v1.0",
+//    baseImage:"ballerina/ballerina-platform:0.980.0"
+//}
+//@docker:CopyFiles {
+//    files:[{source:"/home/saneth/Documents/ballerina/mysql-connector-java-5.1.46.jar",
+//        target:"/ballerina/runtime/bre/lib"}]
+//}
+//
+//@docker:Expose {}
+endpoint http:Listener contentfilterEP {
+    port:9090
+};
+endpoint http:Listener claimvaditadeEP {
+    port:9094
+};
+endpoint http:Listener contentenricherEP {
+    port:9092
+};
+endpoint http:Listener backendEP {
+    port:9093
+};
+//define endpoints for services
+endpoint http:Client validatorEP {
+    url: "http://localhost:9094/validator"
+};
+endpoint http:Client enricherEP {
+    url: "http://localhost:9092/enricher"
+};
+endpoint http:Client clientEP {
+    url: "http://localhost:9093/backend"
+};
+
+
+
+//define the global variables
+public json payload1;
+public json payload2;
+
+//service for the content filter pattern
+service<http:Service> contentfilter bind contentfilterEP {
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/"
+    }
+    contentfilter(endpoint caller, http:Request req) {
+        http:Request filteredreq = req;
+        var jsonMsg = req.getJsonPayload();
+        match jsonMsg {
+            json msg => {
+                //create the student table in the DB
+                var ret = testDB->update(
+                                      "CREATE TABLE student(id INT, name VARCHAR(255), city VARCHAR(255), gender VARCHAR(255))"
+                );
+                handleUpdate(ret, "Create the table");
+                //Error handling for the user inputs
+                if (!check msg.id.toString().matches("\\d+")) {
+                    error err = { message: "student ID containts invalid data" };
+                    http:Response res = new;
+                    res.statusCode = 400;
+                    res.setPayload(untaint err.message);
+                    caller->respond(res) but { error e =>
+                    log:printError("Error sending response", err = e) };
+                } else if (!check msg.name.toString().matches("[a-zA-Z]+")) {
+                    error err = { message: "student Name containts invalid data" };
+                    http:Response res = new;
+                    res.statusCode = 400;
+                    res.setPayload(untaint err.message);
+                    caller->respond(res) but { error e =>
+                    log:printError("Error sending response", err = e) };
+                }else if (!check msg.city.toString().matches("^[a-zA-Z]+([\\-\\s]?[a-zA-Z0-9]+)*$")) {
```

Comment:
Fix formatting

Reply:
Done

Reply:
There are a lot of formatting issues. Can you use spaces and have correct formatting whenever required?